### DISCUSSION TITLE
Unchangeable Quick Filter

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5337] [Help]
![image](https://user-images.githubusercontent.com/24252615/103876724-12ea2600-50e5-11eb-8ba9-04313619b83f.png)

admin comes to the filter by default, but can be changed later.
I want to make the incoming default value unchangeable

**Comment by VictorTomaili** [comment]
Try to add this filter from repository with your logged in user id. 
If you add this filter from front-end side what needs to be unchangeable, you also add a security issue on your project. I guess default value is current user. So you can add a where condition inside repository.cs file > listhandler > override prepareQuery with your logged in user id
